Form/ChangePassword.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Form;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class ChangePassword
+{
+    /**
+     * @Assert\Length(
+     *     min = 6,
```

Comment:
gohja, laat 6 maar staan. We willen ook niet dat mensen "a" als wachtwoord gebruiken.